documentation/src/js_console.md

```diff
@@ -0,0 +1,116 @@
+
+# Forest JavaScript Console
+
+Forest console stands as an alternative to tools like [Curl](https://github.com/curl/curl) or other subcommands found in `forest-cli` for interacting with the Filecoin JSON-RPC API.
+
+## Starting the console
+
+`forest-cli attach` can be used to open a Javascript console connected to your Forest node.
+
+Like for some other `forest-cli` subcommands you will need to pass an admin `--token` given what endpoints you will call.
+
+For a description of different options please refer to the developer documentation CLI page.
+
+## Interactive Use
+
+First start Forest node inside another terminal:
+
+```bash
+./target/release/forest --chain calibnet --encrypt-keystore false
+```
+
+Copy the Admin token printed in the logs and set it to an environment variable, ie:
+
+```bash
+TOKEN="..."
+```
+
+To attach to your Forest node, run `forest-cli` with the `attach` subcommand:
+
+```bash
+./target/release/forest-cli --token $TOKEN attach 
+```
+
+You should now see a prompt and be able to interact:
+
+```                                          
+Welcome to the Forest Javascript console!
+
+To exit, press ctrl-d or type :quit
+> console.log("Forest running on " + chainGetName())
+Forest running on calibnet
+```
+
+You can directly call JSON-RPC API endpoints that have been binded to the console.
+
+For example `Filecoin.ChainGetName` will be binded to the global `chainGetName` function.
+
+### Tips
+
+- Console history is saved in your `~/.forest_history`
+- Use `:clear` to erase session commands before exiting
```

Comment:
Where is it erased? It doesn't seem to erase the console history or clear the screen.

Reply:
It's erasing command of the current session.
Not global history (if you want to erase history from beginning you can wipe `~/.forest_history`.

Reply:
I agree, the wording could be better in the documentation.